API_compatible_README.md

```diff
@@ -4,21 +4,23 @@
 
 **SOURCE_1**: http://redmine.audacious-media-player.org/boards/1/topics/788
 
+### For development
 1. Install dependencies from **SOURCE_1** then clone the repo and install audacious.
-2. Before installing audacious-plugins, edit the file `audacious-plugins/src/scrobbler2/scrobbler.h` to update the following settings on lines L26-L28.  
-  - `SCROBBLER_API_KEY` to the key given on "listenbrainz.org/user/import" page.
-  - `SCROBBLER_URL` to "http://last.fm/2.0/".  
-  Now compile and install the plugins from the instructions given in **SOURCE_1**.
-3. Register an application on musicbrainz with the following callback URL "http://www.last.fm/login/musicbrainz/post"
-4. Update the received client-id and client-secret in config.py of listenbrainz.
-5. Edit the `/etc/hosts` file and add the following entries.
+2. Before installing audacious-plugins, edit the file `audacious-plugins/src/scrobbler2/scrobbler.h` to update the following setting on line L28. This is required only because the local server does not have https support.  
+  - `SCROBBLER_URL` to "http://ws.audioscrobbler.com/2.0/".  
```

Comment:
Hmm. This is a more serious problem than just not having https support on local development.
If the client checks SSL certificates then we will never be able to intercept requests even if the user changes their hosts file. Can you do a survey of a handful of last.fm clients (we've been recommending a number of them in IRC) and see if you can work out which ones use http and which use https. You can make this list on the jira ticket related to the compatibility API. We will then need to check which https clients have certificate checking.
It may be that many of these are open source, and therefore our options are greater (add a native LB submitter, add an option to change host between last.fm/librefm/LB, etc), but we won't know until we look.